From fbcd82040cb67ee14886c6ac4c200cde3662f986 Mon Sep 17 00:00:00 2001
From: Rich Felker <dalias@aerifal.cx>
Date: Mon, 1 Jul 2013 13:50:02 -0400
Subject: [PATCH 2/2] add stubs for additional legacy ether.h functions

these would not be expensive to actually implement, but reading
/etc/ethers does not sound like a particularly useful feature, so for
now I'm leaving them as stubs.
---
 include/netinet/ether.h |  6 +++---
 src/network/ether.c     | 15 +++++++++++++++
 2 files changed, 18 insertions(+), 3 deletions(-)

diff --git a/include/netinet/ether.h b/include/netinet/ether.h
index c5179d5..d64c9ef 100644
--- a/include/netinet/ether.h
+++ b/include/netinet/ether.h
@@ -4,11 +4,11 @@
 #include <netinet/if_ether.h>
 
 char *ether_ntoa (const struct ether_addr *);
-
 struct ether_addr *ether_aton (const char *);
-
 char *ether_ntoa_r (const struct ether_addr *, char *);
-
 struct ether_addr *ether_aton_r (const char *, struct ether_addr *);
+int ether_line(const char *, struct ether_addr *, char *);
+int ether_ntohost(char *, const struct ether_addr *);
+int ether_hostton(const char *, struct ether_addr *);
 
 #endif
diff --git a/src/network/ether.c b/src/network/ether.c
index ac55411..4304a97 100644
--- a/src/network/ether.c
+++ b/src/network/ether.c
@@ -41,3 +41,18 @@ char *ether_ntoa (const struct ether_addr *p_a) {
 	static char x[18];
 	return ether_ntoa_r (p_a, x);
 }
+
+int ether_line(const char *l, struct ether_addr *e, char *hostname)
+{
+	return -1;
+}
+
+int ether_ntohost(char *hostname, const struct ether_addr *e)
+{
+	return -1;
+}
+
+int ether_hostton(const char *hostname, struct ether_addr *e)
+{
+	return -1;
+}
-- 
1.8.3.2

